zipline/src/hostMain/kotlin/app/cash/zipline/EventListener.kt

```diff
@@ -85,6 +85,19 @@ abstract class EventListener {
   ) {
   }
 
+  /**
+   * Invoked when an application load was skipped because the code is stale.
+   *
+   * @param startValue the value returned by [applicationLoadStart] for the start of this call. This
+   *   is null unless [applicationLoadStart] is overridden to return something else.
+   */
+  open fun applicationLoadSkippedNotFresh(
+    applicationName: String,
+    manifestUrl: String?,
```

Comment:
Do we call out what null means for this anywhere? I see it's nullable on other functions too without docs. I'm guessing it's for local?

Reply:
yea this is for local, I was a bit conflicted when I noticed the null, when I was working on this PR: 

1) what if later we'll use this API somewhere else 
2) but where? or later when? 

also because I'm not fully aware of all the implications of changing an API (is it costly, or it's not), so I left it this way 

if changing API (later adding a function in event listener or so) is not a big deal, I'd be happy if the decision is to remove this field, given what we know currently about use case.